core/store/src/main/java/uk/gov/gchq/gaffer/store/StoreProperties.java

```diff
@@ -247,6 +276,7 @@ public String getId() {
      * Set the ID for the StoreProperties
      *
      * @param id the value of the ID
+     * @deprecated the ID should be supplied to the graph library separately
      */
     public void setId(final String id) {
```

Comment:
There are still a few usages of this method. Please can we update them unless they are specifically for the old id logic. For example in StorePropertiesTest.shouldRemovePropertyWhenPropertyValueIsNull  and in FederatedGetSchemaHandlerTest.